cbmc/proofs/IotMqtt_PublishAsync/Makefile

```diff
@@ -0,0 +1,33 @@
+ENTRY=IotMqtt_PublishAsync
+
+# We abstract all the log and concurrency related functions in this proof
+# and assume their implementation is correct
+ABSTRACTIONS += --remove-function-body IotLog_Generic
+ABSTRACTIONS += --remove-function-body _IotMqtt_ProcessSend
+ABSTRACTIONS += --remove-function-body _destroyMqttConnection
+
+OBJS += $(ENTRY)_harness.goto
+OBJS += $(MQTT)/cbmc/proofs/mqtt_state.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_api.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_helper.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_operation.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_serialize.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_validate.goto
+
+# One more than actual number of subscriptions
+SUBSCRIPTION_COUNT_MAX=4
+DEF += -DSUBSCRIPTION_COUNT_MAX=$(SUBSCRIPTION_COUNT_MAX)
+
+# One more than actual number of operations
+OPERATION_COUNT_MAX=4
+DEF += -DOPERATION_COUNT_MAX=$(OPERATION_COUNT_MAX)
+
+LOOP += IotListDouble_RemoveAllMatches.0:$(SUBSCRIPTION_COUNT_MAX)
+LOOP += valid_IotMqttSubscriptionList.0:$(SUBSCRIPTION_COUNT_MAX)
+LOOP += valid_IotMqttOperationList.0:$(OPERATION_COUNT_MAX)
+LOOP += _encodeRemainingLength.0:$(SUBSCRIPTION_COUNT_MAX)
+
+UNWINDING += --unwind 1
+UNWINDING += --unwindset '$(shell echo $(LOOP) | sed 's/ /,/g')'
```

Comment:
This is not needed anymore - CBMC supports multiple `--unwindset` paramaters

Reply:
There are multiple Makefiles on the MQTT proofs that invokes only one `--unwindset` flag in that style. So instead of only changing this one here, I propose to create a separate PR to update all the Makefiles IFF the CBMC version we are using actually supports multiple `--unwindset` flags.

Reply:
Create an issue to track this.

Reply:
Agree.  We have some consistency issues in our Makefiles, our style has matured over the last three weeks.

Reply:
Tracking on issue https://github.com/aws/aws-iot-device-sdk-embedded-C/issues/812